Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM praekeltfoundation/vxsandbox
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get-install.sh npm
+RUN npm install vumigo_v02 lodash
```

Comment:
It might be worthwhile putting these onto one line like:
```sh
RUN apt-get-install.sh npm && npm install vumigo_v02 lodash
```
Docker caches one image per line so this create fewer intermediate images (potentially if we could clean up intermediate build results that aren't needed it'd also make the final image smaller, but that's probably not the case here).